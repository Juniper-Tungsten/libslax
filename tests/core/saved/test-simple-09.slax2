version 1.1;

match _name {
    expr "Hmmmm";
}
