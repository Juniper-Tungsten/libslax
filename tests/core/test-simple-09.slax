version 1.0;

match _name {
    expr "Hmmmm";
}
