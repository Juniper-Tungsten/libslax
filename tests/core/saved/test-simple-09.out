<?xml version="1.0"?>

  fish
  This is 'one'
  This is 'two'
  10
  12
  hand
  yup
  fish
  trout
  this thing
  that thing
  10000

  
    phil
    45
    It worked!!
  

  
    raleigh
    nc
    27614
    1312
  

  7.1R1.4

  Hmmmm


